parts/linux/cloud-init/artifacts/health-monitor.sh

```diff
@@ -6,14 +6,17 @@
 set -o nounset
 set -o pipefail
 
+
 container_runtime_monitoring() {
   local -r max_attempts=5
   local attempt=1
   local -r crictl="${KUBE_HOME}/bin/crictl"
-  local -r container_runtime_name="${CONTAINER_RUNTIME_NAME:-docker}"
-  local healthcheck_command="docker ps"
-  if [[ "${CONTAINER_RUNTIME:-docker}" != "docker" ]]; then
-    healthcheck_command="${crictl} pods"
+  local -r container_runtime_name=$2
+
+  if [[ ${container_runtime_name} == "containerd" ]]; then
+    local healthcheck_command="ctr --namespace k8s.io container list"
```

Comment:
should it be ${crictl}? is the ctr in the current PATH?

Reply:
or remove line 13 as it's not being used

Reply:
yea it's in $PATH. `ctr` is the containerd-native cli that comes with moby-containerd whereas crictl is a more generic cri-cli that we install separately. thus I think using `ctr` is more apt in case for w.e reason we don't install crictl anymore.

Reply:
cool. thanks for the explanation. please remove line 13 then